### PR TITLE
Change Public site page size to 25 and add all counties selected as filter default 

### PR DESCRIPTION
**Pull request by phattya**
# Pull Request Template

## Issue Overview
BAHFA requested a larger page count, so changing to 25. Also updating the filter default to be all counties, since none of our seed data is from the 3 counties that were checked, and it's easier for the user to start with all data shown.

This PR addresses #issue

- [x] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## How Can This Be Tested/Reviewed?
Recommend seeding with
To test `yarn dev` on public & backend core (or `yarn dev:all` if that's the way you like to do it) and check the filter defaults are set as shown here:
image

and check that the page count covers 25 per page ... if you used `yarn db:reseed:with-external:dev` you will see the following
image

Describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration.

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

**Review by isaac-looker — APPROVED** [review]

File: sites/public/src/pages/listings.tsx [hunk 1]
```diff
@@ -58,7 +58,8 @@ export function getServerSideProps() {
       listingsEndpoint: getListingServiceUrl(),
       googleMapsApiKey: runtimeConfig.getGoogleMapsApiKey(),
       // show Bloom counties by default
-      initialSearch: "counties:Alameda,San Mateo,Santa Clara",
+      initialSearch:
+        "counties:Alameda,Contra Costa,Marin,San Francisco,San Mateo,Santa Clara,Solano,Sonoma",
```
Comment (ifranch) [inline sites/public/src/pages/listings.tsx:62]: Should we add Napa as well?

Comment (phattya) [reply sites/public/src/pages/listings.tsx:62]: @ifranch great point! I didn't even notice that Napa wasn't on there (rushing too quickly). It doesn't exist in the Filter modal right now.  @isaac-looker will need to ensure it exists in the new modal build out PR.

Comment (phattya) [reply sites/public/src/pages/listings.tsx:62]: It's funny how we've demoed that a few times and no one has noticed. 😱

Comment (phattya) [reply sites/public/src/pages/listings.tsx:62]: Does it sound ok by you, @ifranch, if we do that in a subsequent PR?

**Review by lisunshiny — APPROVED** [review]

**Pull request merged**